dapp/src/components/shared/Form/FormTokenBalanceInput.tsx

```diff
@@ -28,14 +29,28 @@ export function FormTokenBalanceInput({
     }
   }, [defaultValue, setAmount])
 
+  const handleChange = useCallback<ChangeEventHandler>(
+    (event) => {
+      field?.onChange(event)
+
+      if (!meta.error) {
+        return
+      }
+
+      helpers.setError(undefined)
```

Comment:
```suggestion
if (meta.error) helpers.setError(undefined)
```

Reply:
Used in ea8558141dd662758dbf9a90673a2ce396042a26